gradle.properties

```diff
@@ -1,5 +1,5 @@
 GROUP=app.cash.paparazzi
-VERSION_NAME=1.0.0-SNAPSHOT
```

Comment:
Was this intentionally not set to 0.5.1?

Reply:
Yea, it was set back to SNAPSHOT after deploying the release: https://github.com/cashapp/paparazzi/commit/93a089f1e1fb3ae55e4a6159f4595b744dd0d78a